examples/thoughts/ext/MainPage.js

```diff
@@ -69,7 +62,6 @@ const Thought = (props) => {
       try {
         await createThought({ textMarkdown: textMd.trim(), tagNames })
         history.push('/thoughts') // TODO: Would be cool if this was type checked somehow or if string was coming from the Wasp API.
-        resetForm()
```

Comment:
Swapping the order between `history.push` and `resetForm` would have also fixed the issue (as would adding a check whether the component was mounted before resetting the state). However, there's no need to reset the state before unmounting, so I removed the entire thing.

Reply:
Nice!